ci-operator/step-registry/cucushift/installer/rehearse/aws/ipi/disconnected/private/sts/custom-endpoints/deprovision/cucushift-installer-rehearse-aws-ipi-disconnected-private-sts-custom-endpoints-deprovision-chain.yaml

```diff
@@ -0,0 +1,12 @@
+chain:
+  as: cucushift-installer-rehearse-aws-ipi-disconnected-private-sts-custom-endpoints-deprovision
```

Comment:
If "custom-endpoints" is a must for sts + disconnected clusters, maybe we can delete `custom-endpoints` subfolder from the path?